resources/views/settings/agreement/nda/index.blade.php

```diff
@@ -34,15 +47,15 @@
                                 <div class="col-md-12">
                                     <div class="form-group">
                                         <label for="round_mail_subject">Subject</label>
-                                    <input type="text" name="round_mail_subject" class="form-control" value="{{ $round->{$mailTemplate}['subject'] }}">
+                                    <input type="text" name="round_mail_subject" class="form-control" value="{{ $mailSubject }}">
```

Comment:
You can use it like this as well. 
```suggestion
                                    <input type="text" name="round_mail_subject" class="form-control" value="{{ $round->{$mailTemplate}['subject'] ?? '' }}">
```

Just try it once.